spec/features/admin/loan_flow_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+feature 'loan flow' do
+  let(:division) { create(:division) }
+  let(:user) { create(:person, :with_member_access, :with_password, division: division).user }
+  let!(:loan) { create(:loan, division: division) }
+
+  before do
+    login_as(user, scope: :user)
+  end
+
+  # This should work, but for some reason it fails a lot more often
```

Comment:
This does not make a lot of sense, but specs will fail more often on loan for no reason it seems.